### PR TITLE
feat: autopublish binary scripts improvements.

### PR DESCRIPTION
**Pull request by pantharshit00**
1. Linux auto publishment
2. Nicer UI
3. Env error handling
4. Automatically bumps package.json

added changes requested in #4024

### CODE REVIEW

File: cli/packages/prisma-cli/package.json [hunk 1]
```diff
@@ -48,13 +50,14 @@
   "scripts": {
     "build": "rimraf dist && tsc -d && chmod +x dist/index.js",
     "lint": "tslint src/**/*.ts",
-    "publish-homebrew": "dotenv -e scripts/.env ts-node ./scripts/release-brew.ts",
-    "make-binary": "npm run build && npm run package",
+    "publish-binaries": "dotenv -e scripts/.env ts-node ./scripts/release-binaries.ts",
```
Comment (divyenduz) [inline cli/packages/prisma-cli/package.json]: Make separate commands for publishing binaries, it should be possible to release one without the other as we might want to (say) have beta release on brew but not curl.

File: cli/packages/prisma-cli/package.json [hunk 2]
```diff
@@ -48,13 +50,14 @@
...
+    "binary-osx": "npm run build && npm run package",
+    "binary-linux": "npm run build && npm run package-linux",
```
Comment (divyenduz) [inline cli/packages/prisma-cli/package.json]: Either keep one of the two `binary-osx`, `package` and `binary-linux` as they do the same thing OR move them together in `package.json` as that is more readable.

File: cli/packages/prisma-cli/package.json [hunk 3]
```diff
@@ -48,13 +50,14 @@
...
     "precommit": "lint-staged",
     "prettier": "prettier --single-quote --no-semi --trailing-comma all --write '*.ts' 'src/**/*.ts'",
     "prepublishOnly": "npm run lint && npm run build",
-    "package": "./scripts/cp-assets.sh && pkg package.json --targets node10-macos-x64",
-    "package-linux": "./scripts/cp-assets.sh && pkg package.json --targets node10-linux-x64"
+    "package": "rm -rf ./prisma && ./scripts/cp-assets.sh && pkg package.json --targets node10-macos-x64",
```
Comment (divyenduz) [inline cli/packages/prisma-cli/package.json]: Thanks for adding the remove script, we can also do a "clean up" after publishing operation by removing tar files once the upload is done while packaging.

File: cli/packages/prisma-cli/scripts/release-binaries.ts [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+    )[0].tag_name
+    console.log(`Version to publish: ${stableReleaseVersion}`)
+    await logger(
+      updateVerion(stableReleaseVersion),
+      'Updating package.json to new version',
+      {
+        estimate: 1000,
+      },
+    )
+    await brew(stableReleaseVersion)
```
Comment (divyenduz) [inline cli/packages/prisma-cli/scripts/release-binaries.ts]: Same as above, both operations should not happen with a single command.

**Comment by pantharshit00** [comment]
Closing in favour of #4133

**Pull request closed without merging**